ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/EthFeeHistory.java

```diff
@@ -332,25 +345,93 @@ private List<Wei> getBaseFees(final List<BlockHeader> blockHeaders) {
         .toList();
   }
 
+  private List<Wei> getBlobBaseFees(final List<BlockHeader> blockHeaders) {
+    if (blockHeaders.isEmpty()) {
+      return Collections.emptyList();
+    }
+    // Extend block headers with their first parent
+    List<BlockHeader> extendedBlockHeaders = prependParent(blockHeaders);
+
+    // Calculate base fees for each pair of parent and headers.
```

Comment:
Why do we need to recalculate?

Reply:
We do not save the blobBase fee in the header